bo/technischeressource.go

```diff
@@ -0,0 +1,32 @@
+package bo
+
+import (
+	"github.com/hochfrequenz/go-bo4e/com"
+	"github.com/hochfrequenz/go-bo4e/enum/emobilitaetsart"
+	"github.com/hochfrequenz/go-bo4e/enum/erzeugungsart"
+	"github.com/hochfrequenz/go-bo4e/enum/speicherart"
+	"github.com/hochfrequenz/go-bo4e/enum/technischeressourcenutzung"
+	"github.com/hochfrequenz/go-bo4e/enum/technischeressourceverbrauchsart"
+	"github.com/hochfrequenz/go-bo4e/enum/waermenutzung"
+)
+
+type TechnischeRessource struct {
+	Geschaeftsobjekt
+	TechnischeRessourceId            string                                                            `json:"technischeRessourceId" validate:"required"`                                                                             //Identifikationsnummer einer TechnischeRessource
```

Comment:
ist in c# nullable
https://github.com/Hochfrequenz/BO4E-dotnet/blob/efe49e5d6ed879cc9e0637225812276c512d1d54/BO4E/BO/TechnischeRessource.cs#L32